daml-lf/integration-test-lib/src/main/com/daml/lf/CantonConfig.scala

```diff
@@ -111,4 +113,22 @@ final case class CantonConfig(
       ),
     )
   }
+
+  // Prefer this whenever possible - ledgerIds should be avoided
```

Comment:
Should `ledgerClient` get a comment indicating that it (and so ledger IDs) should be thought of as deprecated?

Is there any value in saying why ledger IDs should be avoided?

Reply:
no sure. 
I think the ledgerId does not really matter anymore.

Reply:
> Should ledgerClient get a comment indicating that it (and so ledger IDs) should be thought of as deprecated?
Perhaps, I'd like Remy's thoughts on keeping both though, it might be easier to switch whatever is using the withoutledger one to the current main, then drop ledger id everywhere in one swoop

> Is there any value in saying why ledger IDs should be avoided?
Honestly, I don't even know what that reason is, someone from canton told me they're near enough deprecated, maybe we need a proper stance on this